docs/getting-started/tutorial-load-dataset.asciidoc

```diff
@@ -3,12 +3,16 @@
 
 This tutorial requires three data sets:
 
-* The complete works of William Shakespeare, suitably parsed into fields. Download
-  https://download.elastic.co/demos/kibana/gettingstarted/shakespeare_6.0.json[`shakespeare.json`].
-* A set of fictitious accounts with randomly generated data. Download
-  https://download.elastic.co/demos/kibana/gettingstarted/accounts.zip[`accounts.zip`].
-* A set of randomly generated log files. Download
-  https://download.elastic.co/demos/kibana/gettingstarted/logs.jsonl.gz[`logs.jsonl.gz`].
+* The complete works of William Shakespeare, suitably parsed into fields
+* A set of fictitious accounts with randomly generated data
+* A set of randomly generated log files
+
+Create a new working directory where you want to download the files. From that directory, run the following commands:
+
+[source,shell]
+curl -O https://download.elastic.co/demos/kibana/gettingstarted/7.x/shakespeare.json
```

Comment:
we can optionally bump this to 8.x, or wait.  either way works for me

Reply:
I'd vote for the forward port.

Reply:
Just copied the files over!  Hopefully we're good now

Reply:
Thanks, just tested things out locally - works great.